pkg/ubasic/ubasic_tests.mk

```diff
@@ -0,0 +1,6 @@
+SRC := tests.c
+NO_AUTO_SRC := 1
+
+MODULE = ubasic_tests
```

Comment:
nit: usually this line is the first in a module's makefile.

Reply:
Fixed